frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.tsx

```diff
@@ -51,7 +93,8 @@ const BreakdownCard: React.FC<DashboardItemProps> = ({
   const ind = top5MetricsStats.findIndex((v) => v.name === 'Others');
```

Comment:
You can use the same constant of 'Others' here.

Reply:
Thanks for pointing this. Actually this is noobaa 'Others' coming from metric, not needed to change. The text is 'Other' only.

Reply:
ack